UndertaleModLib/Models/UndertaleEmbeddedTexture.cs

```diff
@@ -311,62 +298,45 @@ public void Dispose()
     /// </summary>
     public class TexData : UndertaleObject, INotifyPropertyChanged, IDisposable
     {
-        private byte[] _textureBlob;
-        private static MemoryStream sharedStream;
+        private GMImage _image;
 
         /// <summary>
-        /// The image data of the texture.
+        /// The underlying image of the texture.
         /// </summary>
-        public byte[] TextureBlob
-        {
-            get => _textureBlob;
+        public GMImage Image 
+        { 
+            get => _image;
             set
             {
-                _textureBlob = value;
-                OnPropertyChanged();
+                _image = value;
+                OnPropertyChanged(nameof(Image));
```

Comment:
Why is it necessary to provide the name?

Reply:
I forgot the `CallerMemberName` attribute was there, so I've now removed this.